app/models/user_stats.rb

```diff
@@ -43,6 +43,6 @@ def messages_in_discussions_count(date_range = nil)
   end
 
   def organization_exercises
-    organization.book.exercises
+    @organization_exercises ||= organization.exercises
```

Comment:
Why not just caching it in exercises method of organization?

Reply:
Because it's defined with a `has_many` association so it is cached by default.
With the current implementation of this method no memoization is needed at all. I added it to be extra sure in case that implementation changes :stuck_out_tongue:

Reply:
_controversial_